### DISCUSSION TITLE
Responsive UI

### DISCUSSION THREAD

**Discussion by Abhinavxox** [Aashray446/DSA_MADE_EASY#42] [Ideas]
Have you guys focused on the UI more or the functionality?

**Comment by Aashray446** [comment]
We focus on the functionality more as well as the UI, but unfortunately the simulation is not available for smaller devices

**Comment by Abhinavxox** [comment]
But your idea is great about dsa visualisation. Don't you think you should host it and make it scale up?

**Reply by Aashray446** [reply]
Yeah we are first trying to make it bug-free and make it available in mobile also.

**Reply by Abhinavxox** [reply]
Okay have you thought about doing an internship for us?

**Reply by Aashray446** [reply]
Thanks for the opportunity but I am afraid I cannot as I am already occupied with freelancing work.

**Reply by Abhinavxox** [reply]
Its fine but the offer is always there, keep that in mind
